index.js

```diff
@@ -421,13 +421,14 @@ class GithubScm extends Scm {
      * @param  {Object}     config.scmInfo      Information about the repo
      * @param  {String}     config.token        Admin token for repo
      * @param  {String}     config.url          Payload destination url for webhook notifications
+     * @param  {String}     config.actions      Actions for the webhook events
      * @return {Promise}                        Resolves when complete
      */
     async _createWebhook(config) {
-        let action = 'createHook';
+        let action = 'createWebhook';
         const params = {
             active: true,
-            events: ['push', 'pull_request', 'create', 'release'],
+            events: config.actions,
```

Comment:
this change is github specific, right? won't work if we try to pass in the same events for gitlab or bitbucket
might be better to just pass generic hook information

Reply:
Well, I guess this needs some more refactoring. I'll need to make similar changes in`scm-gitlab` and `scm-bitbucket` too.